standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -237,24 +238,28 @@ private void startLocalSegmentStores() throws Exception {
      * @param segmentStoreId id of the SegmentStore.
      */
     private void startLocalSegmentStore(int segmentStoreId) throws Exception {
+        ConfigBuilder<ServiceConfig> serviceConfigConfigBuilder = ServiceConfig.builder()
```

Comment:
Why do we have `ConfigConfig` in the name of this variable? It looks like a typo.

Reply:
Fixed.